File: w09/d02/homework/brainteasers/prime.rb
#get prime number 1-100

prime = [];
i = 1.0;
x = 1.0;
counter = 0;

while x <= 100.0

	while i <= 100.0
	num = x/i
    floor = num.floor
		if num - floor == 0
			counter+=1
		end
	i+=1
	end

	if counter == 2
		prime.push(x.floor)
	end

	counter = 0
	i = 1.0
	x += 1.0
end

puts prime

File: w09/d02/homework/brainteasers/prime.js
//get prime  numbers 1 - 100

var prime = [];
var i = 1;
var x = 1;
var counter = 0;

while(x <= 100){
	while(i <= x - 1){
		var num = x/i;
		if(num.toString().indexOf(".") === -1){
			counter++;
		}
		i++;
	}
	if(counter === 1){
		prime.push(x);
	}
	counter = 0;
	i = 1;
	x++;
}


prime.forEach(function(each){
	console.log(each);
})